be/src/vec/spill/spill_writer.cpp

```diff
@@ -35,13 +35,19 @@ Status SpillWriter::open() {
     return Status::OK();
 }
 
+SpillWriter::~SpillWriter() {
+    if (!closed_) {
+        LOG(WARNING) << "spill writer not closed correctly:\n" << boost::stacktrace::stacktrace();
```

Comment:
not use this way, boost has a static global lock.
Using doris::Status wrapped error stack, you could use it.